neofs-testlib/neofs_testlib/env/env.py

```diff
@@ -1152,7 +1152,7 @@ def prepare(
         size=1024,
         containers=1,
         out="grpc.json",
-        preload_obj=250,
+        preload_obj=100,
```

Comment:
can u explain meaning of this value? If as u said `creating a high number of objects doesn't make sense`, it can be `10` then?

Reply:
What's the difference between 250 and 100 in practice?

Reply:
1m30s to create 250 objects and 30s to create 100 objects. 1m30s can cause some timeouts (which I encountered in this PR) inside tests. So I decreased it a little bit. 10 objects is too small as for me for this scenario, I feel like we can miss something with 10, but this is just feelings. I can decrease up to 2 objects actually.

Reply:
OK for me, but you need to explain this in the commit message.

Reply:
updated the commit message